static/js/tools/map/stat_var_chooser.tsx

```diff
@@ -82,22 +92,60 @@ export function StatVarChooser(): JSX.Element {
         });
     }
   }, [statVar.value]);
+
+  function onSvModalOpened() {
+    if (svHierarchyModalRef.current && svHierarchyContainerRef.current) {
+      svHierarchyModalRef.current.appendChild(svHierarchyContainerRef.current);
+    }
+  }
+
+  function onSvModalClosed() {
+    document
+      .getElementById("explore")
+      .appendChild(svHierarchyContainerRef.current);
+  }
+
   return (
-    <div className="explore-menu-container" id="explore">
-      <DrawerToggle
-        collapseElemId="explore"
-        visibleElemId="stat-var-hierarchy-section"
-      />
-      <StatVarHierarchy
-        type={StatVarHierarchyType.MAP}
-        places={samplePlaces}
-        selectedSVs={[statVar.value.dcid]}
-        selectSV={(svDcid) => {
-          selectStatVar(statVar, display, placeInfo, svDcid);
-        }}
-        searchLabel="Statistical Variables"
-      />
-    </div>
+    <>
+      <div className="d-none d-lg-flex explore-menu-container" id="explore">
+        <DrawerToggle
+          collapseElemId="explore"
+          visibleElemId="stat-var-hierarchy-section"
+        />
+        <div ref={svHierarchyContainerRef}>
+          <StatVarHierarchy
+            type={StatVarHierarchyType.MAP}
+            places={samplePlaces}
+            selectedSVs={[statVar.value.dcid]}
+            selectSV={(svDcid) => {
+              selectStatVar(statVar, display, placeInfo, svDcid);
+            }}
+            searchLabel="Statistical Variables"
+          />
+        </div>
+      </div>
+      <Modal
```

Comment:
this piece of code along with onSvModalOpened and onSvModalClosed seems to be duplicated across scatter, timeline, and stat var explorer - wondering if it would be possible to move the modal code into StatVarHierarchy

Reply:
yes - i was trying to figure out how since there are only slight differences, mostly in handler code. i'll do that consolidation as a follow up..